geopandas/tests/test_overlay.py

```diff
@@ -592,3 +593,113 @@ def test_overlap_make_valid(make_valid):
     else:
         with pytest.raises(ValueError, match="1 invalid input geometries"):
             overlay(df1, df_bowtie, make_valid=make_valid)
+
+
+class TestOverlayWikiExample:
+    def setup_method(self):
+        self.layer_a = GeoDataFrame([{"geometry": box(0, 2, 6, 6)}])
```

Comment:
```suggestion
        self.layer_a = GeoDataFrame(geometry=[box(0, 2, 6, 6)])
```

Can you pass geometries as a list to `geometry` directly instead of dict-based constructor (in all of them)? It is shorter and will simplify some bits below.

Reply:
Sure